Documentation/howto/ovs-with-p4-executables.rst

```diff
@@ -661,6 +661,33 @@ previously configured CONFIG params.
     tdi-portin-id: Port ID for Pipeline in Input Direction
     tdi-portin-id: Port ID for Pipeline in Output Direction
 
+10) Get atrributes for port statistics retrieval::
+
+    $ gnmi-cli get PARAMS
+    $ Example for LINK ports :
+    gnmi-cli get "device:physical-device,name:PORT0,counters" | grep "name\|uint_val" | grep -v "interface\|key\|config\|counters"
+    $ Example for vhost ports :
+    gnmi-cli get "device:virtual-device,name:net_vhost0,counters" | grep "name\|uint_val" | grep -v "interface\|key\|config\|counters"
+    $ Example for TAP ports :
+    gnmi-cli get "device:virtual-device,name:TAP0,counters" | grep "name\|uint_val" | grep -v "interface\|key\|config\|counters"
+
+  .. note::
+
+    Port stats can be retrieved for the ports that are configured through GNMI CLI.
+    These ports can be of type LINK/VHOST/TAP type. gnmi-cli by default outputs
+    the data in yang model, hence format the output using the grep comamnd for
```

Comment:
so the output is formatted using the grep command to display port statistics

Reply:
Done